### DISCUSSION TITLE
[SOLVED] Notification and Reading Pane (Web Browser)

### DISCUSSION THREAD

**Discussion by joseashb** [martinrotter/rssguard#569] [Ideas]
Hello mate,

Switched from QuiteRSS to RSS Guard. It is nice and working very well. I was wondering if whether you could add the below it would be really nice.

Feature 1: Notification shows list of news
Feature 2: Reading Pane always visible
Feature 3: Allow manual sorting of Feeds list
Feature 4: Resizable icons (16x16,32x32,48x48)
Feature 5: Fix issue on HiDPI (4k) screen with scaling (150%). (Had to set DPI to "System (enhanced)")
Feature 6: Start from unread news when click on a feed

Also I saw that in the reading pane, the images gets distorted:

![image](https://user-images.githubusercontent.com/47283948/148099868-cdff319d-e629-4504-b6c9-c10deeee7e58.png)

Thanks for the good work mate. Happy new year @martinrotter and team

**Comment by akinokonomi** [comment]
> Also I saw that in the reading pane, the images gets distorted:

Please post the link to feed you see that in.

**Reply by joseashb** [reply]
https://www.xda-developers.com/feed

**Reply by akinokonomi** [reply]
> https://www.xda-developers.com/feed

See no distortion in their recent articles when using nudus-light/nudus-dark skins, please try them instead (available in latest release https://github.com/martinrotter/rssguard/releases/tag/4.1.1). 
You can also enable "Force dark look" option, for a complete dark theme.

**Reply by martinrotter** [reply]
Also see no visual problems here -> note that the feed is of low quality.

**Comment by martinrotter** [comment]
1. https://github.com/martinrotter/rssguard/issues/499
2. https://github.com/martinrotter/rssguard/issues/424
3. https://github.com/martinrotter/rssguard/issues/302
4. Fixed RN - https://github.com/martinrotter/rssguard/commit/fd1355e3c093d1cc68ae834ddc735c3041e5784e
5. Report BUG ticket for this with screenshots of what exactly is wrong.
6. Just sort your articles from unread column - it gets remembered, feature is already there.

Distortion, like @akinokonomi wrote, provide the link to feed. This is likely caused by shitty feed quality.
